jenkins/Jenkinsfile

```diff
@@ -78,6 +78,7 @@ pipeline {
                         sh 'sudo chown -R jenkins:jenkins $WORKSPACE'
                         checkout scm
                         sh 'docker login -u "$CANDLEPIN_QUAY_BOT_USR" -p "$CANDLEPIN_QUAY_BOT_PSW" quay.io'
+                        sh 'python jenkins/generate_custom_yaml.py'
```

Comment:
We should try to avoid adding any testing logic in the pipeline script. It should only contain infrastructure-related commands (docker login/logout, git checkout, etc.). There are multiple levels where this script could be run:
1) jenkins/rspec-tests.sh
2) docker/test
3) cp-test
I would strongly prefer to add this script in the candlepin-base image and run it from cp-test, since that is the closest to the actual tests running, and anyone that would download/build the candlepin-base image to run tests locally would have the cloud tests running there too (while if we only left this in the jenkins pipeline script, they wouldn't run)